.travis.yml

```diff
@@ -20,12 +20,12 @@ script:
 after_success:
   - travis-cargo doc-upload
   - if [ "${TRAVIS_PULL_REQUEST_BRANCH:-$TRAVIS_BRANCH}" != "master" ] && [ "$TRAVIS_RUST_VERSION" == "nightly" ]; then
```

Comment:
You sir, are a genious.

The `&&` i wanted to keep so that subsequent steps don't run if earlier ones fail.